Firmware/Tupfile.lua

```diff
@@ -1,7 +1,65 @@
 
 tup.include('build.lua')
 
-boarddir = 'Board/v3.3'
+-- Switch between board versions
+boardversion = tup.getconfig("BOARD_VERSION")
+if boardversion == "" then boardversion = "v3.4" end
```

Comment:
It is dangerous to have a default like this. It hasn't happened yet, but it's concievable that the config of 1 board blows up another board. Better make sure people make it explicit.